internal/storage/resource_storage_pool.go

```diff
@@ -353,6 +353,8 @@ func (_ StoragePoolModel) ComputedKeys(driver string) []string {
 			"lvm.vg_name",
 			"lvm.thinpool_name",
 		}
+	case "lvmcluster":
+		// TODO
```

Comment:
Are there any computed keys required before merging?

Reply:
Added the two keys that I think we should be expecting now